dex-operator/cmd/dex-operator/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"context"
+	"runtime"
+
+	stub "github.com/operator-framework/operator-sdk-samples/dex-operator/pkg/stub"
+	sdk "github.com/operator-framework/operator-sdk/pkg/sdk"
+	k8sutil "github.com/operator-framework/operator-sdk/pkg/util/k8sutil"
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+
+	"github.com/sirupsen/logrus"
+)
+
+func printVersion() {
+	logrus.Infof("Go Version: %s", runtime.Version())
+	logrus.Infof("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
+	logrus.Infof("operator-sdk Version: %v", sdkVersion.Version)
+}
+
+func main() {
+	printVersion()
+
+	sdk.ExposeMetricsPort()
+
+	resource := "auth.example.com/v1alpha1"
+	kind := "Dex"
+	namespace, err := k8sutil.GetWatchNamespace()
+	if err != nil {
+		logrus.Fatalf("Failed to get watch namespace: %v", err)
```

Comment:
This doesn't seem like the right way to used structures logging. You need to first initialize a logger with a config. See: https://github.com/coreos/dex/blob/6ef8cd512fb9e075bf7d70ec3fe74a8397eadde4/cmd/dex/serve.go#L66